R/writeManifest.R

```diff
@@ -56,56 +56,38 @@ writeManifest <- function(appDir = getwd(),
     appFiles <- explodeFiles(appDir, appFiles)
   }
 
-  quartoInfo <- inferQuartoInfo(
+  appMetadata <- appMetadata(
```

Comment:
My preference would be to avoid variable shadowing like you've done here; is that common practice in other R code?

Reply:
You tell me: https://github.com/rstudio/rsconnect/blob/main/R/deployApp.R#L331-L333 😛 

But seriously, I'll keep thinking about this name as I continue to refactor this code.